spyder/dependencies.py

```diff
@@ -177,6 +177,10 @@
      'package_name': "numpydoc",
      'features': _("Improve code completion for objects that use Numpy docstrings"),
      'required_version': NUMPYDOC_REQVER},
+    {'modname': "packaging",
+     'package_name': "packaging",
+     'features': _("Core utilities for handling version numbers of Python packages"),
```

Comment:
```suggestion
     'features': _("Compare version numbers of Python packages"),
```

Simplify this description a bit.

Reply:
Done and rebased